integration_test/cloudbuild-integration-test.yaml

```diff
@@ -362,8 +362,10 @@
 
 - name: ${_CLOUD_SDK_IMAGE}
   id: publish-image
-  args: [ 'bash', 'integration_test/scripts/publish-images.sh', '${_IMAGEPROJECT}', "${_PUBLISH}"]
+  args: [ 'bash', 'integration_test/scripts/publish-images.sh', '${_IMAGEPROJECT}', "${_PUBLISH}", "$REPO_NAME"]
```

Comment:
What does REPO_NAME resolve to in this case?

Reply:
$REPO_NAME contains "githubUser_githubRepo" if repository is stored by user and just "githubRepo" if stored by organization